src/main/java/com/gluonhq/substrate/target/DarwinTargetConfiguration.java

```diff
@@ -61,9 +63,16 @@ String getAdditionalSourceFileLocation() {
         return answer;
     }
 
+    @Override
+    List<String> getCommonLinkLibraries() {
+        List<String> defaultLinkFlags = new ArrayList<>(super.getCommonLinkLibraries());
+        defaultLinkFlags.add("-lextnet");
+        return defaultLinkFlags;
+    }
+
     private static final List<String> macoslibs = Arrays.asList("-lffi",
             "-lpthread", "-lz", "-ldl", "-lstrictmath", "-llibchelper",
-            "-ljava", "-lnio", "-lzip", "-lnet", "-ljvm", "-lobjc", "-lstdc++",
+            "-ljava", "-lnio", "-lzip", "-lnet", "-ljvm", "-lobjc",
```

Comment:
that means -ljava etc are in the list twice?

Reply:
yes, that was a known issue. We might need to do a proper refactoring of all these libraries.

Reply:
Can you file an issue for this so that we keep track of it?

Reply:
Done